### DISCUSSION TITLE
Cookiecutter folks talk about Copier

### DISCUSSION THREAD

**Discussion by yajo** [copier-org/copier#945] [General]
I was looking for articles talking about Copier, just out of curiosity. I found out this one from cookiecutter folks: https://www.cookiecutter.io/article-post/cookiecutter-alternatives

I loved to see it because the comparison is very good and the article is very friendly! 😊

Thanks, cookiecutter friends! ❤

**Comment by pawamoy** [comment]
Oh, the Copier part is a bit short and hard to understand IMO. It does not convey the clear advantage of using Copier: projects are kept in sync with templates' evolution. Anyway, nice to see it listed ^^

**Reply by bittner** [reply]
And, to be fair, the last paragraph is simply just an invention to make the description of Copier more "balanced".

> However, disadvantages include that this tool is newer, so there’s not as large of a community to engage with, or templates to choose from. In addition, it’s under active development, so it’s not as stable as Cookiecutter. 

The point is, the Cookiecutter project now moves so slowly that it's a pain to push PRs and issues forward. We spent years to get some PRs merged, and some maintainers had nothing else to do than reiterate that "we should pay them" (or the project) for developing features. – If that is "maturity" I'd rather have a project that is "under active development", instead. :smirk: 

Great that there is Copier! It feels so much more lightweight and powerful. :rocket:

**Reply by pawamoy** [reply]
And to be kind, I understand their position. They spent countless hours working on Cookiecutter, offering it for free to the world, so I think it's fair that they ask for some sponsorship. Reviewing PRs and maintaining merged code is still a lot of work. Unfortunately it seems that people, and more importantly companies, still see free, maintained projects as granted, and they only pay when they get access to private features in return (or support, or similar).
